Classes/common/touchdb/ChangeTracker/TDURLConnectionChangeTracker.m

```diff
@@ -103,8 +109,10 @@ - (BOOL)start
 
 - (void)clearConnection
 {
-    [self.connection cancel];
-    [self.connection unscheduleFromRunLoop:[NSRunLoop currentRunLoop] forMode:NSRunLoopCommonModes];
+    if(self.task.state != NSURLSessionTaskStateCompleted){
+        [self.task cancel];
+    }
+    self.task = nil;
     self.connection = nil;
```

Comment:
Why is this line still here if the other connection things are gone? I also don't see the removal of the connection property.